.gitignore

```diff
@@ -23,6 +23,7 @@ dump/*
 
 settings*.json
 *conn.txt
+*settings.js
```

Comment:
This is because you might want to have `local-db-settings.js` in your project directory and not commit it. Broadly, although we have `../ForumCredentials`, the intended way to start developing on this repo is to run with your settings in the project directory.